Wrong line number for empty-comment message

<!--
  Hi there! Thank you for discovering and submitting an issue.

  Before you submit this, make sure that the issue doesn't already exist
  or if it is not closed.

  Is your issue fixed on the preview release?: pip install pylint astroid --pre -U

-->

### Steps to reproduce
1. create a file with empty comment, for instance:

```
""" test"""
#
```

2. run Pylint with the empty-comment extension  #3870

    for example in command line with a file named `temp.py`:

`pylint --load-plugins=pylint.extensions.empty_comment temp.py`

3. Watch Pylint output in the console.

### Current behavior
You get the following message output:
`temp.py:1:0: R2044: Line with empty comment (empty-comment)`

with line number `1`.


### Expected behavior
You would expect line number `2`, i.e with output:
`temp.py:2:0: R2044: Line with empty comment (empty-comment)`


### pylint --version output
pylint 2.6.1-dev1
astroid 2.5.0
Python 3.7.7 (default, May  6 2020, 11:45:54) [MSC v.1916 64 bit (AMD64)]

running on Windows 10 64-bit

--- a/ChangeLog
+++ b/ChangeLog
@@ -1,6 +1,10 @@
 ------------------
 Pylint's ChangeLog
 ------------------
+* Bug fix for empty-comment message line number.
+
+  Closes #4009
+
 * Only emit `bad-reversed-sequence` on dictionaries if below py3.8
 
   Closes #3940

--- a/pylint/extensions/empty_comment.py
+++ b/pylint/extensions/empty_comment.py
@@ -49,7 +49,7 @@ class CommentChecker(BaseChecker):
                 line = line.rstrip()
                 if line.endswith(b"#"):
                     if not is_line_commented(line[:-1]):
-                        self.add_message("empty-comment", line=line_num)
+                        self.add_message("empty-comment", line=line_num + 1)
 
 
 def register(linter):

--- a/tests/extensions/test_empty_comment.py
+++ b/tests/extensions/test_empty_comment.py
@@ -28,7 +28,7 @@ def test_comment_base_case(linter):
     for msg in msgs:
         assert msg.symbol == "empty-comment"
         assert msg.msg == "Line with empty comment"
-    assert msgs[0].line == 1
-    assert msgs[1].line == 2
-    assert msgs[2].line == 4
-    assert msgs[3].line == 6
+    assert msgs[0].line == 2
+    assert msgs[1].line == 3
+    assert msgs[2].line == 5
+    assert msgs[3].line == 7

